src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/CustomDashedView.swift

```diff
@@ -0,0 +1,93 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class CustomDashedView: UIView {
+
+	enum Mode {
+		case add, modify
+	}
+	
+	// MARK: - Internal
+
+	@IBOutlet weak var label: ENALabel!
+	@IBOutlet weak var icon: UIImageView!
+	
+	@IBInspectable var cornerRadius: CGFloat = 15 {
+		didSet {
+			layer.cornerRadius = cornerRadius
+			layer.masksToBounds = cornerRadius > 0
+		}
+	}
+	@IBInspectable var dashWidth: CGFloat = 2
+	@IBInspectable var dashColor: UIColor = .enaColor(for: .riskNeutral)
+
+	/// Dash pattern - dash length
+	@IBInspectable var dashLength: CGFloat = 5
+	/// Dash pattern - gap length
+	@IBInspectable var betweenDashesSpace: CGFloat = 5
+
+	var tapHandler: (() -> Void)?
+
+	private var dashBorder: CAShapeLayer?
+	private lazy var tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(onTap))
+
+	class func instance(for mode: Mode) -> CustomDashedView {
+		let nibName = String(describing: Self.self)
+		// swiftlint:disable:next force_cast
+		let view = UINib(nibName: nibName, bundle: nil).instantiate(withOwner: nil, options: nil)[0] as! CustomDashedView
```

Comment:
please use a guard and throw an error or something similar. But do not force cast.

Reply:
If the xib isn't present we end up in a fatal error. Same outcome then force unwrap.
Why do we have xibs at first place? Good question…